Model/MissingPropertyMetadataException.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace Sonata\DoctrineORMAdminBundle\Model;
+
+class MissingPropertyMetadataException extends \LogicException
+{
+    public function __construct($class, $property)
+    {
+        parent::__construct(sprintf(
+            'No metadata found for property `%s::$%s`.'
+            . ' Please make sure your Doctrine mapping is properly configured.',
+            $class,
+            $property
+        ));
+    }
+}
```

Comment:
Please configure your IDE to always add a newline at end of file, this will make the red pictogram disappear.